### PR TITLE
{bio}[dummy/] T-Coffee v11.00.8cbe486

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
ca184.camp.thecrick.org - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/f587ab65bb0c50552e729cf60a5d24a4 for a full test report.

**Comment by boegel** [comment]
@verdurin better use `--force` when testing a PR? ;)

**Comment by verdurin** [comment]
@boegel I just built it a minute ago, but yes, yes...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
ca184.camp.thecrick.org - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2c29a49e63257c2fc9f0fceea4c4aba5 for a full test report.

File: easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock='Binary'
+
+name = "T-Coffee"
```
Comment (verdurin) [inline easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb]: Nitpicky, I know, but shouldn't these be single quotes, for consistency?

Comment (verdurin) [inline easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb]: Otherwise, looks good.

Comment (boegel) [inline easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb]: @verdurin I personally try to consistently use single quotes for words (no spaces) and double qoutes for 'sentences' (strings incl. spaces), but I don't think it's worth blocking a PR over.

There are other issues here though, if this is merged as is, the `develop` is going to fail on the automatic style tests that were enabled recently (cfr. #2506).

See the output of `eb --from-pr 4709 --check-style`, or just close/open this PR to retrigger Travis.

Comment (verdurin) [inline easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb]: @boegel - ah. I thought such things would be reported here.

Comment (boegel) [inline easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb]: They should be, but Travis gave the green light on this PR before the style checks were enabled.

**Comment by shahzebsiddiqui** [comment]
I was able to build this from your easyconfig file, thanks @verdurin can we get this merged, looks like a few style changes need to be fixed.

**Comment by verdurin** [comment]
@pescobar some style fixes still required:
```
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:6:10: E225 missing whitespace around operator
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:22:121: E501 line too long (153 > 120 characters)
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:24:18: W299 trailing whitespace
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:25:1: E101 indentation contains mixed spaces and tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:25:1: W191 indentation contains tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:26:1: W191 indentation contains tabs
/home/travis/build/easybuilders/easybuild-easyconfigs/easybuild/easyconfigs/t/T-Coffee/T-Coffee-11.00.8cbe486_linux_x64.eb:30:1: E101 indentation contains mixed spaces and tabs
```

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**